src/figuresClasses.test.ts

```diff
@@ -116,7 +116,7 @@ describe('Method "getArea" of class "Circle"', () => {
     const circle = new Circle('red', 6);
 
     expect(circle.getArea())
-      .toBe(113.09);
```

Comment:
can we change that?

Reply:
@annak1104 @Taras2705 No, the tests do not need to be changed.